packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -80,6 +80,7 @@ const ESGlobalKeys = [
     'Symbol',
     // 'SyntaxError', // Reflective
     // 'TypeError', // Reflective
+    'trustedTypes',
```

Comment:
❤️

Reply:
It was the only way to avoid having it remapped, as far as my knowledge on the near-membrane code could help me. Any other way to prevent it from being remapped ?

Reply:
that is the thing, you should remap it, and distort it but only after creating the default policy for the iframe right before remapping everything.

Reply:
here is my hunch on why this is wrong: if you attempt to use the defaultPolicy object from global after the iframe is detached, it will blow up, because there is no active window behind it. it works for eval because eval is an intrinsic, but probably not for other DOM APIs. That is a hunch, but I don't see a test for that here, let's try it. Basically, inside the sandbox, something like `eval(trustedTypes.defaultPolicy. createScript('1'))`, and that might throw because the DOM API fails due to the iframe being disconnected, while something like `eval(1)` might succeed just fine.